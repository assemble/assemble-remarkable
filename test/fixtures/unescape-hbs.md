# Templates

> Should fix escaped handlebars templates

This is a {{template}}
This is a {{template foo="bar"}}

This is a {{> partial }}

This is a block helper
{{#each foo as |bar|}}
  {{bar}}
{{/each}}
