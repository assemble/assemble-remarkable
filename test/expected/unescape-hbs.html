<h1>Templates</h1>
<blockquote>
<p>Should fix escaped handlebars templates</p>
</blockquote>
<p>This is a {{template}}
This is a {{template foo="bar"}}</p>
<p>This is a {{> partial }}</p>
<p>This is a block helper
{{#each foo as |bar|}}
{{bar}}
{{/each}}</p>
